package.nls.json

```diff
@@ -50,5 +50,15 @@
     "containerApps.openAcrBuildLogs": "Open Azure Container Registry Build Logs",
     "containerApps.deployWorkspaceProject.containerAppName": "When deploying from a local workspace project, the name of the target container app to deploy to.",
     "containerApps.deployWorkspaceProject.containerAppResourceGroupName": "When deploying from a local workspace project, the name of the target container app's resource group.",
-    "containerApps.deployWorkspaceProject.containerRegistryName": "When deploying from a local workspace project, the name of the Azure Container Registry to use for storing and building images."
+    "containerApps.deployWorkspaceProject.containerRegistryName": "When deploying from a local workspace project, the name of the Azure Container Registry to use for storing and building images.",
+    "containerApps.deploymentConfiguration": "A saved deployment configuration used for quickly redeploying a workspace project to a container app.",
+    "containerApps.deploymentConfiguration.label": "The label used when displaying or referencing the configuration. Personalize this to best suit your needs.",
+    "containerApps.deploymentConfiguration.type": "The deployment strategy type for this configuration.",
+    "containerApps.deploymentConfiguration.dockerfilePath": "The relative path from the project's root directory to the Dockerfile.",
+    "containerApps.deploymentConfiguration.srcPath": "The relative path from the project's root directory to the deployment folder.",
+    "containerApps.deploymentConfiguration.envPath": "The relative path from the project's root directory to the environment variables file (.env).",
+    "containerApps.deploymentConfiguration.resourceGroup": "The name of the target container app's resource group.",
```

Comment:
Yeah, I added the "name" just because it's not clear otherwise what the field property needs (is it the resource ID, the name, etc.).

It is usually populated most of the time though, so I could see the argument being that we don't need a super descriptive description since the value we populate already indicates what it is.